proto/store.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "grpc.store";
+option csharp_namespace = "Momento.Protos.Store";
+
+package store;
+
+service Store {
+  rpc Get(_GetRequest) returns (_GetResponse) {}
+  rpc Set(_SetRequest) returns (_SetResponse) {}
+  rpc Delete(_DeleteRequest) returns (_DeleteResponse) {}
+}
+
+message _GetRequest {
+  bytes key = 1;
+  string store_id = 2;
```

Comment:
we don't need a `store_id` for client facing protos. We will need the `store_name` I think but I am not sure because in the cache get/set/delete apis we don't pass the cache_name.

@cprice404 could you comment on this?

Reply:
That rides in a header on cache APIs. That does not mean that is what we should do here, but that's how it works there. I'd personally prefer it to be in the proto, but if Chris has other thoughts I'm open to them.

Reply:
tbh I'd prefer it to be in the proto too. But I feel like someone must have had some motivation for putting the cache name in the header originally. @kvcache ?

If @kvcache doesn't remember a compelling reason for why it was in the header for cache then I'm #TeamProto

Reply:
There were compelling reasons but they no longer exist. Putting it in the message is better in this case, and most cases.

Reply:
okay https://github.com/momentohq/client-protos/pull/277/commits/fff5d408ea9d3d7d16bc46e3359ec5087051d19a